### PR TITLE
Project Labels are managed with issue templates and PR labeler action

### LINKED ISSUE #998: Find a replacement for the Labeler GitHub Action
**Which component**:
GitHub actions

**Describe the bug**
The project is currently using an external GitHub action to identify new issues with a `triage` label automatically. This action can be easily automated using the GitHub [issue template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) built-in feature.

We need to remove the [label GitHub action](https://github.com/bitnami-labs/sealed-secrets/blob/main/.github/workflows/label.yaml) and include the `triage` label automatically in the [templates](https://github.com/bitnami-labs/sealed-secrets/tree/main/.github/ISSUE_TEMPLATE).

### PR DESCRIPTION
**Pull request by agarcia-oss**
Signed-off-by: Alfredo Garcia <algarcia@vmware.com>

**Description of the change**

The Pull Requests include the following changes:

- Added the `triage` label automatically with the issue templates
- Removed the un-maintained labeler action
- Included the PR auto [labeler](https://github.com/marketplace/actions/labeler) Action from GitHub. 

The last point needs to be tested since the labels must be applied only once the PRs are created. 

**Possible drawbacks**
After considering the options, is best not to include the auto labeler Action. This Action requires a `pull_request_target `[integration](https://github.com/marketplace/actions/labeler#create-workflow). This kind of integration is not recommended due to [security concerns](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/). 

**Applicable issues**

- fixes #998

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+triage:
+- any: ['*']
```
Comment (josvazg) [inline .github/labeler.yml:2]: Should we add EOL here?
```suggestion
- any: ['*']

```

File: .github/workflows/label.yaml [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
     steps:
-      - name: Label issues/prs
-        uses: andymckay/labeler@1.0.4
-        with:
-          add-labels: "triage"
-          ignore-if-labeled: true
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/labeler@v4.0.2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (josvazg) [inline .github/workflows/label.yaml:14]: I like EOLs
```suggestion
        repo-token: "${{ secrets.GITHUB_TOKEN }}"

```

Comment (josvazg) [reply .github/workflows/label.yaml:14]: Note DCO does not like suggestions committed directly in the GH ui

**Review by josvazg — DISMISSED** [review]
LGTM except for a couple of EOLs

**Review by alvneiayu — DISMISSED** [review]
LGTM

**Review by josvazg — APPROVED** [review]

**Review by alvneiayu — APPROVED** [review]
LGTM

**Pull request merged**